manticore/ethereum/__init__.py

```diff
@@ -626,6 +626,11 @@ def transactions(self, state_id=None):
         state = self.load(state_id)
         return state.platform.transactions
 
+    def human_transactions(self, state_id=None):
+        """ Transactions list for state `state_id` """
```

Comment:
If id is None and there is more than a single running state it raises an exception. (or it should)